Sming/compiler/ld/eagle.app.v6.cpp.include.ld

```diff
@@ -0,0 +1,217 @@
+/*  
```

Comment:
> btw, are there any advantages of the espressif bootloader? Would it make sense to to support rboot only to simplify the framework?

Can you post this as discussion in the issues? If no one really needs the espressif bootloader we get rid of it and can easily simplify and unify our build system :)